basic-assignments-5-start/src/app/app.component.html

```diff
@@ -0,0 +1,7 @@
+
+<div class="topnav">
+    <br>
+    <a class="active" href="#">Home</a>
+    <a routerLink="/star-wars-movies">star-wars-movies</a>
```

Comment:
Why do you use href once inside a tag and then you use routerLink. choose one of them